Chore Create Default Users

Cheza-Dzabala opened the pull request:
#### What does this PR do?
Creates seeders for default Users and Posts

#### Description of Task to be completed?
Create default seeders so that tests can run and sign in on existing accounts


#### How should this be manually tested?

1. Clone the repository
2. Run the ` npm install ` command
3. Run the ` npm run test ` command
4. Manually check your Postgres __TEST__ database for the following:  

- seeded user accounts: 

a. John Doe (johndoe@gmail.com)
b. Jane Doe (janedoe@gmail.com)

- seeded posts:

a. Description: 
> This is a demo post one. This post does the awesome post things. No post is more awesomer than this.

b. Description: 
> This is a demo post two! This post does the awesome post things. No post is more awesomest than this.

#### Any background context you want to provide?

N /A

#### What are the relevant pivotal tracker stories?
##### #168752605

#### Screenshots (if appropriate)
N/A
#### Questions:
N/A

Cheza-Dzabala committed:
Chore Create Default Users

- Created a seeder for default users
- Merged the postTests.js and getPostsTest.spec.js into one file
- Updated package.json file to run seeders in test database

[starts #168752605]

Cheza-Dzabala committed:
CH Add default users seeder

- Updated travis.yml file to add admin environment variables

[#168752605]

alainmateso commented:
Please add the migration to add the image column to the posts table!

sabin18 commented on __test__/__posts__/getPostsTest.spec.js:
```diff
@@ -9,30 +9,14 @@ chai.use(chaiHttp);
-        done();
-      });
-  });
+  // it('Should not get any post', (done) => {
```
you should remove the commented code here .

Cheza-Dzabala committed:
CH add default seeders / Apply feedback

- Removed all commented code
- Added an image migration on Posts table
- Added image field to posts model

[#168752605]

ngireric123 commented on __test__/__posts__/getPostsTest.spec.js:
```diff
@@ -9,30 +9,14 @@ chai.use(chaiHttp);
-  });
+  // it('Should not get any post', (done) => {
+  //   chai.request(app)
+  //     .get('/api/v1/posts')
```
remove these comments

alainmateso reviewed (approved):
It looks great!

ngireric123 reviewed (commented):
it is ok now
